deployment/v3/mosip/bqatsdk/values.yaml

```diff
@@ -0,0 +1,5 @@
+bqatsdk:
+  ## This is url of bqatsdk lib accessed from within cluster.  BqatSDK may be hosted outside the cluster
+  ## in which case change the URL accordingly.
+  zippedLibUrl: http://artifactory.artifactory/artifactory/libs-release-local/biosdk/bqatsdk-lib.jar
+  bioapiImpl: io.bqat.sdk.services.impl.BqatQualitySdkService
```

Comment:
Please remove values.yaml as the same values has been added to helm charts values.yaml